src/App.js

```diff
@@ -1,29 +1,13 @@
-import logo from './logo.svg';
-import './App.css';
+// import React from 'react';
```

Comment:
Great job @ijeomaodinko 🤝

- [ ] [Optional] Kindly remove commented code from your project as this is not regarded as a good practice.